integration/src/test/java/integration/HomePageToCompetency.java

```diff
@@ -0,0 +1,48 @@
+package integration;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.WebDriver;
+import pages.BandsPage;
+import pages.CompetencyPage;
+import pages.HomePage;
+import pages.JobRolesPage;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class HomePageToCompetency {
+    WebDriver driver = RunCucumberTest.getDriver();
```

Comment:
Without a visibility keyword, this `driver` is visible to other classes in the package. If it shouldn't be, use `private`.